packages/features/bookings/lib/getBookingResponsesSchema.test.ts

```diff
@@ -565,7 +565,7 @@ describe("getBookingResponsesSchema", () => {
       expect(parsedResponses.data).toEqual({
         email: "test@test.com",
         name: "test",
-        testPhone: "+919999999999",
```

Comment:
Probably worth keeping this test and adding a new one for the Australian number.  I'm sure there was a reason they were testing this

Reply:
I have tried it and it's OK. I don't think there's a need to separately add test cases specifically for Australian phone numbers on cal.com.